rmgpy/rmg/main.py

```diff
@@ -42,6 +42,7 @@
 import csv
 import gc
 import copy
+from CoolProp.CoolProp import PropsSI
```

Comment:
I wonder if there is a way to avoid a dependency to CoolProp in the main module, by encapsulating it into a module related to solvation?